SPECS/rubygem-bigdecimal/rubygem-bigdecimal.spec

```diff
@@ -1,19 +1,19 @@
-%global debug_package %{nil}
 %global gem_name bigdecimal
+
 Summary:        BigDecimal provides arbitrary-precision floating point decimal arithmetic
-Name:           rubygem-bigdecimal
-Version:        3.1.2
```

Comment:
Any specific reason we're upgrading to v3.1.6? Alternatively, we can get rid of the separate spec, and get v3.1.5 with ruby v3.3.0

Reply:
We want the latest version for 3.0 and that's just what's latest at this moment. At least that's how I see our 3.0 effort.:)